pkg/resource/logging-secret/alloy/logging-secret.yaml.template

```diff
@@ -0,0 +1,36 @@
+alloy:
```

Comment:
1. Do we really need to override all of this just to set some config?
2. Why do we have to use templates? This reminds me of the PMO stuff we have and I'm not sure I like it :( 
3. Could you add comments explaining what the template do?

Reply:
1. Yes we need those override to make alloy work as a logging agent
2. Using template save the complexity of defining all the yaml structure as Go struct, its then easier to update those values later on.
3. I'll add that